lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -53,6 +53,19 @@ import {
 } from "../account/AccountInfo";
 import * as CacheHelpers from "../cache/utils/CacheHelpers";
 
+function parseServerErrorNo(
+    serverResponse: ServerAuthorizationCodeResponse
+): string | undefined {
+    const errorCodePrefix = "code=";
+    const errorCodePrefixIndex =
+        serverResponse.error_uri?.lastIndexOf(errorCodePrefix);
+    return errorCodePrefixIndex && errorCodePrefixIndex >= 0
+        ? serverResponse.error_uri?.substring(
+              errorCodePrefixIndex + errorCodePrefix.length
```

Comment:
Are we sure it will always be the last item? Would it be more robust to take all numbers between "AADSTS" and ":" in the message?

Reply:
`error_uri` looks like a better option in terms of performance. It is not mutable AFAIK.